File: 06-Fibonacci/fibonacci.js
function fibonacci(n) {
  let primaryValues = [0, 1]; 
  let result = 0;

  if (n === 1) {
    console.log(primaryValues[0]);
  } else if (n >= 2) {
    console.log(primaryValues[0]); 
    console.log(primaryValues[1]);
  }

  for (let i = 2; i < n; i++) { 
    result = primaryValues[0] + primaryValues[1]; 
    primaryValues[0] = primaryValues[1];
    primaryValues[1] = result;
    console.log(result);
  }
}


fibonacci(10);


//Recursive Solution

// function FibonacciRecursiveVersion(n) {
//   if (n === 1) {
    
//   }
// }

File: 06-Fibonacci/fibonacci.py
def fibonacci(n):
    primaryValues = [0,1]
    result = 0
    
    if n == 1:
        print(0)
    elif n >= 2:
        print(0) 
        print(1)   
    
    for value in range(2,n):
        result = primaryValues[0] + primaryValues[1]
        primaryValues[0] = primaryValues[1]
        primaryValues[1] = result
        print(result) 
    
fibonacci(10)    
        